fints/utils.py

```diff
@@ -55,7 +56,7 @@ def parse(self, lines):
         finsegs = self.grab_financial_instrument_segments(clauses)
         # Third: Extract financial instrument data
         for finseg in finsegs:
-            isin, name, market_price, price_symbol, price_date, pieces = (None,)*6
+            isin, name, market_price, price_symbol, price_date, pieces, einstandspreis = (None,)*7
```

Comment:
Maybe we should call it ``acquisitionprice`` here as well for less confusion? ;)